fedcred/common.py

```diff
@@ -134,21 +140,24 @@ def get_sts_creds(arn):
 
 def write_credentials(profile, creds):
     aws_creds_path = '%s/.aws/credentials' % (os.path.expanduser('~'),)
-    config = configparser.ConfigParser()
-    creds_folder = os.path.dirname(aws_creds_path)
-    if not os.path.isdir(creds_folder):
-        os.makedirs(creds_folder)
+    bash_creds_path = '%s/.aws_credentials' % (os.path.expanduser('~'),)
```

Comment:
What's the purpose of writing the same information in 2 locations? Every AWS SDK should now reference `~/.aws/credentials`, what's the use case for also having environment variables?